src/components/App/WorkspaceSwitcher.vue

```diff
@@ -35,6 +35,7 @@ export default {
   },
   methods: {
     ...mapActions("auth", ["setActiveWorkspace"]),
+    ...mapActions("generic", ["setUserSwitchedWorkspace"]),
```

Comment:
not required if re-rendering is implemented right?

Reply:
So, I removed it first, but then it started acting buggy, right? So, I wanted to see which change of mine was bringing that about, but couldn't test it. So waiting.